src/entity/Reviewable.java

```diff
@@ -1,4 +1,74 @@
 package entity;
 
-public interface Reviewable {
+import entity.review.Review;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+
+public class Reviewable {
```

Comment:
Maybe change it to an abstract class, since there cannot be an instance of a reviewable?